WooCommerce/Classes/Authentication/Store Creation/Free Trial/FreeTrialFeatures.swift

```diff
@@ -7,44 +7,50 @@ struct FreeTrialFeatures {
+            static let growYourBusiness = NSLocalizedString("Grow your business", comment: "Title of features on the Free Trial Summary screen")
+            static let hassleFreeStoreOwnership = NSLocalizedString(
+                "Hassel-free store ownership",
+                comment: "Title of features on the Free Trial Summary screen"
+            )
+        }
         enum Feature {
-            static let premiumThemes = NSLocalizedString("Premium themes", comment: "Premium Themes title feature on the Free Trial Summary Screen")
-            static let unlimitedProducts = NSLocalizedString("Unlimited products", comment: "Unlimited products title feature on the Free Trial Summary Screen")
-            static let shippingLabels = NSLocalizedString("Shipping labels", comment: "Shipping labels title feature on the Free Trial Summary Screen")
-            static let ecommerceReports = NSLocalizedString("Ecommerce reports", comment: "Ecommerce reports title feature on the Free Trial Summary Screen")
-            static let paymentOptions = NSLocalizedString("Multiple payment options",
-                    comment: "Multiple payment options title feature on the Free Trial Summary Screen")
-            static let subscriptions = NSLocalizedString("Subscriptions & product kits",
-                    comment: "Subscriptions & product kits title feature on the Free Trial Summary Screen")
+            static let themes = NSLocalizedString("Premium themes", comment: "Premium Themes title feature on the Free Trial Summary Screen")
+            static let storeManagement = NSLocalizedString("Store management tools", comment: "Store management title feature on the Free Trial Summary Screen")
+            static let codeFree = NSLocalizedString("Code-free design and editing", comment: "Code-free title feature on the Free Trial Summary Screen")
+            static let seo = NSLocalizedString("Built-in SEO", comment: "SEO title feature on the Free Trial Summary Screen")
             static let advertising = NSLocalizedString("Social advertising", comment: "Social advertising title feature on the Free Trial Summary Screen")
-            static let marketing = NSLocalizedString("Email marketing", comment: "Email marketing title feature on the Free Trial Summary Screen")
+            static let email = NSLocalizedString("Automated customer emails", comment: "Email title feature on the Free Trial Summary Screen")
+            static let hosting = NSLocalizedString("Best-in-class hosting", comment: "Hosting title feature on the Free Trial Summary Screen")
+            static let backups = NSLocalizedString("Automated site backups", comment: "Site backup title feature on the Free Trial Summary Screen")
             static let support = NSLocalizedString("24/7 support", comment: "24/7 support title feature on the Free Trial Summary Screen")
```

Comment:
Should this be changed to "24/7 priority support"?

Reply:
Great catch! Updated in d0e869cf6b158c4eeb74ef041428719f39ff8142.